### DISCUSSION TITLE
to use kafkametricsreceiver,  how to define a list of broker?

### DISCUSSION THREAD

**Discussion by Dionysusio** [open-telemetry/opentelemetry-collector-contrib#4885] [Q&A]
eg:  config file in receiver/kafkametricsreceiver/testdata/config.yaml

`receivers:
  kafkametrics:
    **brokers: 10.10.10.10:9092**
    protocol_version: 2.0.0
    scrapers:
      - brokers
      - topics
      - consumers
    auth:
      tls:
        ca_file: ca.pem
        cert_file: cert.pem
        key_file: key.pem
    collection_interval: 5s
`

the example is one broker,  config like this:
`brokers: 10.10.10.10:9092`

i want to define a list broker, like this?
`brokers: 10.10.10.10:9092,10.10.10.11:9092`

after  i config like that, i found my kafka cluster is missing some data, when i config two broker, there is missing consumers....
and config one kafka broker is no problem.

please help me !  what is the problem?  and what is the right list broker config? 
thanks,

**Comment by bogdandrutu** [comment]
I messed up with issue/discussion, sorry.

cc @dshomoye

**Comment by dshomoye** [comment]
The `broker` key expects a YAML array list,
For example:
```yml
brokers: 
  - 10.10.10.10:9092
  - 10.10.10.11:9092
```
